prepare-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+set -eux
+
+version=$1
+
+if ! echo "$version" | grep -q "^[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+$"; then
+  echo "argv[1] needs to be a version, in x.y.z format."
+  exit 1
+fi
+
+git fetch
+git checkout origin/main
+git branch -D "release-v$version" || true
+git checkout -b "release-v$version"
+
+sed -i '/^version = ".*"$/s/^.*/version = "'"$version"'"/' Cargo.toml
```

Comment:
Rather than deal with a briddle `sed`, maybe we could add the `toml-cli` to our flake and use that on this instead.

Reply:
Nice!